OPCB/functions/missions/spawn_city_ins.sqf

```diff
@@ -1,30 +1,27 @@
+params ["_centerobj"];
 
-	_centerobj = _this select 0;
-
-	_houses = nearestObjects [_centerobj, ["house"], 500] select { count ( _x buildingPos -1 ) > 4 };
-	
-	_defendergroup = creategroup [east,true];
-
-	for "_i" from 0 to count _houses -1 do {
-		_item = _houses select _i;
-		_chance = floor (random 5);
-
-		if(_chance == 1) then
-		{
-			_positions = [_item] call BIS_fnc_buildingPositions;
-			if(count _positions >2)
-			then
-			{
-				_pos = selectrandom _positions;
-				_unit =	 _defendergroup createUnit [selectrandom OPCB_unitTypes_inf_ins, _pos, [], 1, "NONE"];
-				_unit disableAI "MOVE";
-				_unit setunitpos "UP";
-				sleep 0.3;
-			};
+_houses = nearestObjects [_centerobj, ["house"], 500] select { count ( _x buildingPos -1 ) > 4 };
```

Comment:
I'm still worried about this returning 0 in a smaller town or a different map or something... But let's see how this goes...